pylearn2/scripts/tutorials/convolutional_network/tests/test_convnet.py

```diff
@@ -14,16 +14,19 @@
 
 import os
 from pylearn2.testing import skip
-from pylearn2.testing import no_debug_mode
 from pylearn2.config import yaml_parse
 
 
 def test_convolutional_network():
-
+    """Test smaller version of convolutional_network.ipynb"""
     skip.skip_if_no_data()
     yaml_file_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
                                                   '..'))
     save_path = os.path.dirname(os.path.realpath(__file__))
+    # Escape potential backslashes in Windows filenames, since
+    # they will be processed when the YAML parser will read it
+    # as a string
+    save_path.replace('\\', r'\\')
```

Comment:
This works on unix too, and may even fix cases where there is a litteral backslash in a path.